sql/src/main/java/io/crate/metadata/table/StaticTableInfo.java

```diff
@@ -59,6 +61,10 @@ public StaticTableInfo(RelationName ident, ColumnRegistrar columnRegistrar, List
         this(ident, columnRegistrar.infos(), columnRegistrar.columns(), primaryKey);
     }
 
+    public StaticTableInfo(RelationName ident, ColumnRegistrar columnRegistrar, String... primaryKey) {
```

Comment:
should it be smth like `ColumnRegistrar<?>`

Reply:
I don't think we win anything here if we add a generic type. I would not bind StaticTableInfo to T of ColumnRegistrar and it is also does not make sense to bind only the constructor. It would not increase type safety from my perspective.